e2e/live/end-to-end-test.sh

```diff
@@ -759,11 +759,11 @@ echo "Testing apply/prune after migrate"
 echo "kpt live apply e2e/live/testdata/migrate-case-1b"
 cp -f e2e/live/testdata/migrate-case-1a/Kptfile e2e/live/testdata/migrate-case-1b
 ${BIN_DIR}/kpt live apply e2e/live/testdata/migrate-case-1b > $OUTPUT_DIR/status
-assertContains "namespace/test-rg-namespace unchanged"
```

Comment:
@mortent The expected output is `configured` here. `unchanged` and `configured` are used interchangeably some times. Please confirm this behavior.